packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -116,13 +156,12 @@ export default class ApplePayButtonStrategy implements CheckoutButtonStrategy {
             );
         }
 
-        document.body.classList.add('apple-pay-supported');
-
         const button = document.createElement('div');
 
-        button.setAttribute('class', buttonClassName);
         button.setAttribute('role', 'button');
         button.setAttribute('aria-label', 'Apple Pay button');
+        Object.assign(button.style, getApplePayButtonStyle(styleOption));
```

Comment:
It might be better to add a condition to run this code only if `styleOption` is available.

Reply:
We do not need to add a condition here to check if the `styleOption` is present or not, because if it is not, we will get the default style configuration